deploy_openshift_aio.sh

```diff
@@ -47,8 +48,20 @@ rm -r $OC_DIR $OC_TARFILE
 cd $SCRIPT_DIR
 oc version
 
-title "Deploying OpenShift AIO cluster"
+title "Initializing OpenShift AIO cluster"
 oc cluster up --service-catalog=true --public-hostname=$(hostname -f)
+oc cluster down
+
+title "Reconfiguring OpenShift AIO cluster"
+if ! grep "pods-per-node" $PATH_NODE_CONFIG; then
+  sed -i -e "s/kubeletArguments:/kubeletArguments:\n  pod-per-node:\n  - \"0\"/" $PATH_NODE_CONFIG
```

Comment:
This should probably read "pods-per-node".

Reply:
Thanks for the eyes, have an update on that one kept mentally swapping node-config and pods-per-core